.github/workflows/go-develop.yml

```diff
@@ -0,0 +1,210 @@
+        run: |
+          echo "`cat ${{github.workspace}}/VERSION`-${{ env.REF_TAG }}" > VERSION
+
+      - name: Get VERSION
+        run: |
+          echo "VERSION=`cat ${{github.workspace}}/VERSION`" >> $GITHUB_ENV
+
+      - name: Debug VERSION
+        run: echo ${{ env.VERSION }}
+
+      - name: Build services (go build only)
+        run: make services
+
+      - name: Build orb agent (go build only)
+        if: steps.filter.outputs.agent == 'true'
+        run: make agent_bin
+
+      - name: Go unit tests
+        run: make test
+
+      - name: Build UI
+        if: steps.filter.outputs.ui == 'true'
+        run: make ui
+
+  package:
+    needs: build
+    runs-on: ubuntu-latest
+    # if this is a push into one of our main branches (rather than just a pull request), we will also package
+    if: github.event_name != 'pull_request'
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            agent:
+              - 'agent/**'
+              - 'cmd/agent/**'
+            fleet:
+              - 'fleet/**'
+              - 'cmd/fleet/**'
+            policies:
+              - 'policies/**'
+              - 'cmd/policies/**'
+            sinks:
+              - 'sinks/**'
+              - 'cmd/sinks/**'
+            sinker:
+              - 'sinker/**'
+              - 'cmd/sinker/**'
+            ui:
+              - 'ui/**'
+
+      - name: Get short commit hash to a variable
+        id: commit_hash
+        run: |
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+
+      - name: Get version to a variable
+        id: version
+        run: |
+          echo "::set-output name=version::$(cat VERSION)"
+
+      - name: Commit orb-ui-live on orb-live repo develop
+        if: env.BRANCH_NAME == 'develop'
+        run: |
+          git config --global user.email "${{secrets.GH_ORB_EMAIL}}"
+          git config --global user.name "${{secrets.GH_ORB_USER}}"
+          git config --global credential.helper cache
+          git clone -b develop https://${{secrets.GH_ORB_USER}}:${{secrets.GH_ORB_ACCESS_TOKEN}}@github.com/ns1labs/orb-live.git          
+          cd orb-live
+          cp -rf ../ui .
+          cp -rf ../VERSION .
+          echo "${{ steps.commit_hash.outputs.sha_short }}" > COMMIT_HASH          
+          git add .
+          git commit -m "[NS1 Orb Bot] Update Orb Live UI for develop"
+          git push origin develop
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build service containers
+        if: env.BRANCH_NAME == 'develop'
+        run: make dockers
+
+      - name: Build orb-agent
+        shell: bash
+        if: steps.filter.outputs.agent == 'true' || env.BRANCH_NAME == 'develop'
```

Comment:
remove and others below that check for branch name 'develop', which is always true